detector.go

```diff
@@ -70,8 +70,18 @@ func (c *DetectConfig) process(done []Buildpack) ([]Buildpack, []BuildPlanEntry,
 		}
 		run := t.(detectRun)
 		if len(run.Output) > 0 {
-			c.Logger.Debugf("======== Output: %s ========", bp)
-			c.Logger.Debug(string(run.Output))
+			outputLogf := c.Logger.Debugf
+
+			switch run.Code {
+			case CodeDetectPass, CodeDetectFail:
+			case -1:
+				outputLogf = c.Logger.Infof
+			default:
+				outputLogf = c.Logger.Infof
+			}
+
+			outputLogf("======== Output: %s ========", bp)
+			outputLogf(string(run.Output))
 		}
 		if run.Err != nil {
 			c.Logger.Debugf("======== Error: %s ========", bp)
```

Comment:
Should do the same thing here for error logs that we did above with `outputLogf`?

Reply:
I had that, and then I undid it. I actually kind of think `run.Err` being anything other than `nil` should perhaps _always_ go to `info`. If you agree, I'll make that change in this PR

Reply:
>  I actually kind of think run.Err being anything other than nil should perhaps always go to info

Hmm, I think we should stick toggling debug/info based on the exit code. `/bin/detect` might log to stderr or invoke a tool that does, even if there is no error per se.

Reply:
Fair enough. I pushed that change.